uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/manage-leave-requests.component.js

```diff
@@ -254,36 +254,43 @@ define([
     }
 
     /**
-     * Loads the managees and calls loadLeaveRequests()
+     * Loads the leave requests for the given data type
      *
+     * @param {String} dataType - which data to load, table or filter or both
```

Comment:
the name is misleading, here you are really talking about sections (table, filters, etc). Data type hints that there are acttualy different types of data related to leave request